public/app/features/visualization/data-hover/DataHoverView.tsx

```diff
@@ -89,6 +93,24 @@ export const DataHoverView = ({ data, rowIndex, columnIndex, sortOrder, mode, he
     displayValues.sort((a, b) => arrayUtils.sortValues(sortOrder)(a.value, b.value));
   }
 
+  return { displayValues, links };
+}
+
+export const DataHoverView = ({ data, rowIndex, columnIndex, sortOrder, mode, header, padding = 0 }: Props) => {
```

Comment:
only change here is optional `padding` param at end, rest of diff is just logic extraction to exportable fn.